src/model/kaniRunner.ts

```diff
@@ -159,6 +169,13 @@ function executeKaniProcess(
 				error: error,
 			};
 
+			// Send output to diagnostics and return if there is an error in stdout
+			// this means that the command could not be executed.
+			if(stdout.includes("error") && stderr.includes("error"))
```

Comment:
Is this the test that checks whether Kani crashed as opposed to failing a harness?

If so, I'm not sure I agree. Kani might crash before printing any output at all. Also 'error' is a perfectly legitimate string that can appear in stdout/stderr without a crash, sometimes a file might be called `error.rs` or it might appear in the name of a harness.

Would it work if you instead check for a particular string that Kani is supposed to print out, rather than one that it is not supposed to print out? My suggestion is: check for the string `VERIFICATION:-` which CBMC prints out right at the end. If you see that string, Kani hasn't crashed.

Honestly a better idea would be for Kani to have a set of known return codes, and if you don't see any of them, then assume a crash. For example, CBMC returns 0 on success, 10 if a harness fails, 6 when it crashes, etc. We're also thinking about a structured output for Kani so that there's no need to parse strings. But for now, I think checking for `VERIFICATION:- SUCCESSFUL` or `VERIFICATION:- FAILED` should be adequate.

Reply:
Can do! Thanks for the idea, will work that in here.